src/Microsoft.AspNetCore.WebHooks.Receivers.AzureAlert/Extensions/AzureAlertMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.AspNetCore.WebHooks;
+using Microsoft.AspNetCore.WebHooks.Internal;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Alert WebHooks in an <see cref="IMvcBuilder" />.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class AzureAlertMvcBuilderExtensions
+    {
+        /// <summary>
+        /// Add Azure Alert WebHook configuration and services to the specified <paramref name="builder"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder" /> to configure.</param>
+        /// <returns>The <paramref name="builder"/>.</returns>
+        public static IMvcBuilder AddAzureAlertWebHooks(this IMvcBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            AzureAlertServiceCollectionSetup.AddAzureAlertServices(builder.Services);
+
+            return builder.AddWebHooks();
+        }
+
+        /// <summary>
+        /// Add Azure Alert WebHook configuration and services to the specified <paramref name="builder"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder" /> to configure.</param>
+        /// <param name="setupAction">
+        /// An <see cref="Action{WebHookOptions}"/> to configure the provided <see cref="WebHookOptions"/>.
+        /// </param>
+        /// <returns>The <paramref name="builder"/>.</returns>
+        public static IMvcBuilder AddAzureAlertWebHooks(this IMvcBuilder builder, Action<WebHookOptions> setupAction)
```

Comment:
FYI these methods as well as `WebHookOptions` go away in #187. Just here in case this PR is easier to review first.